os/board/rtl8720e/src/component/soc/amebalite/fwlib/ram_km4/ameba_ota_ram.c

```diff
@@ -480,5 +480,41 @@ int OTA_Image2SignatureCheck(uint32_t input_addr)
 	return TRUE;
 }
 
+/* Secure Key Derive Function structure */
+typedef struct {
+	unsigned char *password;
+	size_t plen;
+	unsigned char *salt;
```

Comment:
Could you please add description for these arguments as well

Reply:
Done